terragrunt/modules/docs-rs/web-server.tf

```diff
@@ -1,88 +1,92 @@
+locals {
+  web_domain = "web.${var.domain}"
+}
+
 resource "aws_security_group" "web" {
-  vpc_id = var.vpc_id
+  vpc_id = var.cluster_config.vpc_id
   name   = "docs-rs-web"
 }
 
-# TODO: configure web app
-# module "web" {
-#   source         = "../shared/modules/ecs-app"
-#   cluster_config = data.terraform_remote_state.shared.outputs.ecs_cluster_config
+module "web" {
+  source         = "../ecs-app"
+  cluster_config = var.cluster_config
 
-#   env  = "prod"
-#   name = "docs-rs-web"
-#   repo = "rust-lang/docs.rs"
+  env  = "dev"
+  name = "docs-rs-web"
+  repo = "rust-lang/docs.rs"
 
-#   cpu                  = 256
-#   memory               = 512
-#   tasks_count          = 1
-#   platform_version     = "1.4.0"
-#   ephemeral_storage_gb = 40
+  cpu                  = 256
+  memory               = 512
+  tasks_count          = 1
+  platform_version     = "1.4.0"
+  ephemeral_storage_gb = 40
 
-#   environment = {
-#     DOCSRS_PREFIX               = "/tmp"
-#     DOCSRS_STORAGE_BACKEND      = "s3"
-#     DOCSRS_LOG                  = "docs_rs=debug,rustwide=info"
-#     RUST_BACKTRACE              = "1"
-#     DOCSRS_STATIC_CLOUDFRONT_ID = "${aws_cloudfront_distribution.static.id}"
-#   }
+  environment = {
+    DOCSRS_PREFIX               = "/tmp"
+    DOCSRS_STORAGE_BACKEND      = "s3"
+    DOCSRS_LOG                  = "docs_rs=debug,rustwide=info"
+    RUST_BACKTRACE              = "1"
+    DOCSRS_STATIC_CLOUDFRONT_ID = "${aws_cloudfront_distribution.static.id}"
+  }
 
-#   secrets = {
-#     DOCSRS_GITHUB_ACCESSTOKEN = "/prod/docs-rs/github-access-token"
-#   }
+  secrets = {
+    DOCSRS_GITHUB_ACCESSTOKEN = "/prod/docs-rs/github-access-token"
```

Comment:
I think that we should have a `/staging` namespace now that we're spinning up staging environments. Should we create a GitHub issues for this?

Reply:
This should certainly not be in a prod namespace. However, I think we're not going to have multiple environments in the same account so perhaps we should just drop this completely?

Reply:
Oh that's a good point! :+1: